CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
-0.3.1
-=====
+# yet-another-logger
+
+## Unreleased
```

Comment:
```suggestion
## 0.4.0
```

Reply:
We may also consider adding a date. I find it helpful when changelings also list the date of a release.